packages/css/test/__snapshots__/css.test.js.snap

```diff
@@ -129,7 +130,7 @@ exports[`css composition with objects 1`] = `
   color: red;
 }
 
-@media (min-width:420px) {
+@media (min-width: 420px) {
```

Comment:
the output regarding some whitespaces has changed, this might be visible in a bunch of updated snapshots